### PR TITLE
graphene 1.2.6 (new formula)

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: Library/Formula/graphene.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
```
Comment (DomT4) [inline Library/Formula/graphene.rb]: It'd be nice to run `"make", "check"` here as well.

Comment (tschoonj) [inline Library/Formula/graphene.rb]: @DomT4 I tried but there are two failures.

I submitted an upstream issue at https://github.com/ebassi/graphene/issues/41

Comment (DomT4) [inline Library/Formula/graphene.rb]: Looks like they added a fix & released a new version. If that works, feel free to update, add the `make check` bit and we should be golden.

**Comment by tschoonj** [comment]
Just pushed the new 1.2.6 release and added the `make check` to the formula.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b8b6fca8ebc058f9d1d8e7bfe66b671e11ccdcfc. Thanks @tschoonj!

**Pull request closed without merging**